templates/web/base/report/display.html

```diff
@@ -41,23 +41,14 @@
 [% INCLUDE 'report/banner.html' %]
 [% INCLUDE 'report/_main.html' %]
 
-[% IF problem.duplicate_of %]
-  <div>
-    <h4>[% loc('This report is a duplicate') %]</h4>
-    <p>[% loc("Please leave updates on the original report:") %]</p>
-    <ul class="item-list">
-      [% INCLUDE 'report/_item.html' item_extra_class = 'item-list__item--with-pin item-list--reports__item--selected' problem = problem.duplicate_of %]
-    </ul>
-  </div>
-  [% SET shown_form = 1 # To stop users leaving updates on duplicate reports. %]
-[% END %]
-
 [% TRY %][% INCLUDE 'report/_message_manager.html' %][% CATCH file %][% END %]
 [% INCLUDE 'report/display_tools.html' %]
 [% TRY %][% INCLUDE 'report/sharing.html' %][% CATCH file %][% END %]
 [% INCLUDE 'report/updates.html' %]
 
-[% IF NOT shown_form %]
+[% IF problem.duplicate_of %]
```

Comment:
The main thing I was suggesting was to put the "Please leave updates on the original report" message down where you’d normally type an update in. Because, if I want to leave an update, I'm gonna scroll down there and totally ignore whatever lovely helpful text you’ve put further up the page.

So that’s the "Please leave updates…" bit.

But as for the rest of the "This report is a duplicate" – it makes sense to have that fairly high up, so there’s more chance of you seeing it when the page first loads. Its current position (below the report description and council ref, if it exists, but above the comments) is probably high enough for now.